src/Controller/PageController.php

```diff
@@ -37,7 +37,7 @@ final class PageController extends AbstractController
     public function __construct(
         ExceptionListener $exceptionListener,
         PageServiceManagerInterface $pageServiceManager,
-        CmsManagerSelectorInterface $cmsSelector,
+        CmsManagerSelectorInterface $cmsSelector
```

Comment:
We did not have this loaded on the container, so we didn't catch this error (only for php <= 7.4)